recipe/run_test.bat

```diff
@@ -0,0 +1,8 @@
+cd data
+
+ncgen -o in.nc in.cdl
+
+ncap2 -O -v -s 'erf_one=float(gsl_sf_erf(1.0f));print(erf_one,"%g")' in.nc foo.nc
```

Comment:
@czender it did not crash, but I have no idea how to make `.bat` files to print the value. I did test locally though and I got `0.842701`. See https://ci.appveyor.com/project/ocefpaf/nco-feedstock/build/1.0.56#L1617

I guess we are all set! With a new release I'll update this and merge.

:tada: conda `nco` on windows :tada: